imap_processing/ena_maps/ena_maps.py

```diff
@@ -527,8 +507,64 @@ def project_pset_values_to_map(
                 raise NotImplementedError(
                     "Only PUSH and PULL index matching methods are supported."
                 )
+
             self.data_dict[value_key] += pointing_projected_values
 
+    def __repr__(self) -> str:
+        """
+        Return a string representation of the map.
+
+        Returns
+        -------
+        str
+            String representation of the map.
+        """
+        return f"{self.__class__} Map)"
```

Comment:
Doesn't this return `RectangularSkyMap Map)`? (duplicate Map and extra parentheses) I'd suggest removing this and letting this return the default class repr.

Reply:
Will do! I'm having the subclasses define reprs anyway, so that they can be a bit more specific (e.g. `"HealpixSkyMap with nside=32, nested=False"`) Just added the Healpix repr.